src/promptflow/promptflow/azure/_restclient/flow/_azure_machine_learning_designer_service_client.py

```diff
@@ -12,7 +12,7 @@
 
```

Comment:
Nit: update the rest client changelog:
![image](https://github.com/microsoft/promptflow/assets/7776147/149a5e45-83f1-4249-89fa-917017d6f5e7)

Reply:
updated